### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by kozross** [openlierox/openlierox#790]
As I understand it, the license file there covers libraries. Is it your intent that the whole game is licensed under the LGPL?

**Comment by pelya** [comment]
Yes, that was the intent, you may take any part of the code and make a
shared library from it, then use it in your closed-source project, as long
as you publish modifications to this library.

**Comment by kozross** [comment]
What about assets?

**Comment by albertz** [comment]
The in-game pictures, levels, sounds, etc are not covered and the situation is unclear. Some of it was created by the original LieroX author Jason Boettcher. As he released LieroX under zlib, I guess this also applies to the assets created by him. Some of the assets might have been copied from the original Liero. Most of the levels and their graphics have been created by the community and it's not really possible find out where they got their data from.

We tried to get OLX into Debian once and this unclear situation was the reason it was not accepted. You might still find the discussion in their mailing list archive. Also [here](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=445850) and [here](http://www.openlierox.net/forum/index.php?topic=13509.0). However, this also cannot really be resolved. One solution was to try to select a subset of the levels or so, but even this was too hard and nobody cared enough to start this. Also, (O)LX without most of the mods and levels isn't really the same game. And you need the levels etc to play with other people online. But OLX would download the levels then automatically if they are missing.

The menu graphics were created by our team and are fine.
